tools/lsp/wasm_main.rs

```diff
@@ -11,6 +11,7 @@ mod properties;
 mod semantic_tokens;
 mod server_loop;
 mod util;
+mod wasm_preview;
```

Comment:
Don't we want to keep this gated by the preview feature?

Reply:
If ever we went to have a lightweight LSP wasm binary.

There is already a `preview` feature and with this patch it doesn't compile on wasm without it because that module uses dependencies that are not enabled.